config_flag: false # flag for manual setting of rqt gui parameters (dynamic_reconfigure)

#m1_x:
#  id: ['ankleHRCEML']
#  left: false # flag for left or right limb
#  wrist: false # flag for wrist device
#  c0: [2.181] # coulomb friction constant of joint [Nm]
#  c1: [2.850] # viscous friction constant of joint [Nm/s]
#  c2: [0] # friction square root term
#  i_sin: [0.47] # inertia for sine term (0.84 with previous joint limit)
#  i_cos: [0.22] # inertia for cosine term (0.23 with previous joint limit)
#  t_bias: [-0.349] # theta bias term and angle from vertical position to lower joint limit [rad]
#  force_sensor_scale_factor: [0.074] # scale factor for force calibration
#  force_sensor_offset: [1500] # calibration offset for force sensor
#  ff_ratio: [0.74] # scale factor for feedforward compensation
#  kp: [1.08] # proportional term of PID controller
#  ki: [0.17] # integral term of PID controller
#  kd: [0] # derivative term of PID controller
#  vel_thresh: [6.0] # velocity threshold for using dynamic friction compensation [deg/s]
#  tau_thresh: [0.2] # torque threshold for using static friction compensation (not implemented) [Nm]
#  lowpass_cutoff_freq: [30.0] # cutoff frequency for lowpass filter for interaction torque, angle and velocity [Hz]
#  motor_torque_cutoff_freq: [50.0] # cutoff frequency for lowpass motor torque command filter [Hz]
#  tick_max: [60.0] # counter for integral term reset [s]
#  tracking_offset: [63] # center of range of motion [deg]
#  tracking_df: [73] # max range of motion angle [deg]
#  tracking_pf: [53] # min range of motion angle [deg]
#  mvc_df: [-1] # MVCT in dorsiflexion [Nm]
#  mvc_pf: [1] # MVCT in plantarflexion [Nm]
#  muscle_count: [4] # number of muscles for EMG recordings

m1_y:
  id: ['ankleHRCEML']
  left: false # flag for left or right limb
  wrist: false # flag for wrist device
  c0: [1.6808] # coulomb friction constant of joint [Nm]
  c1: [2.2137] # viscous friction constant of joint [Nm/s]
  c2: [0] # friction square root term
  i_sin: [0.89] # inertia for sine term
  i_cos: [0.25] # inertia for cosine term
  t_bias: [-0.349] # theta bias term and angle from vertical position to lower joint limit [rad]
  force_sensor_scale_factor: [0.074] # scale factor for force calibration
  force_sensor_offset: [1500] # calibration offset for force sensor
  ff_ratio: [0.6] # scale factor for feedforward compensation
  kp: [1.08] # proportional term of PID controller
  ki: [0.17] # integral term of PID controller
  kd: [0] # derivative term of PID controller
  vel_thresh: [6.0] # velocity threshold for using dynamic friction compensation [deg/s]
  tau_thresh: [0.2] # torque threshold for using static friction compensation (not implemented) [Nm]
  lowpass_cutoff_freq: [30.0] # cutoff frequency for lowpass interaction torque, angle and velocity filter [Hz]
  motor_torque_cutoff_freq: [50.0] # cutoff frequency for lowpass motor torque command filter [Hz]
  tick_max: [60.0] # counter for integral term reset [s]
  tracking_offset: [63] # center of range of motion [deg]
  tracking_df: [83] # max range of motion angle [deg]
  tracking_pf: [43] # min range of motion angle [deg]
  mvc_df: [-1] # MVCT in dorsiflexion [Nm]
  mvc_pf: [1] # MVCT in plantarflexion [Nm]
  muscle_count: [4] # number of muscles for EMG recordings

m1_a:
  id: ['ankleSPES']
  left: false # flag for left or right limb
  wrist: false # flag for wrist device
  c0: [1.67] # coulomb friction constant of joint [Nm]
  c1: [2.66] # viscous friction constant of joint [Nm/s]
  c2: [0] # friction square root term
  i_sin: [0.9] # inertia for sine term
  i_cos: [0.16] # inertia for cosine term
  t_bias: [-0.349] # theta bias term and angle from vertical position to lower joint limit [rad]
  force_sensor_scale_factor: [0.074] # scale factor for force calibration
  force_sensor_offset: [1500] # calibration offset for force sensor
  ff_ratio: [0.72] # scale factor for feedforward compensation
  kp: [1.08] # proportional term of PID controller
  ki: [0.17] # integral term of PID controller
  kd: [0] # derivative term of PID controller
  vel_thresh: [6.0] # velocity threshold for using dynamic friction compensation [deg/s]
  tau_thresh: [0.2] # torque threshold for using static friction compensation (not implemented) [Nm]
  lowpass_cutoff_freq: [30.0] # cutoff frequency for lowpass interaction torque, angle and velocity filter [Hz]
  motor_torque_cutoff_freq: [50.0] # cutoff frequency for lowpass motor torque command filter [Hz]
  tick_max: [60.0] # counter for integral term reset [s]
  tracking_offset: [45] # center of range of motion [deg]
  tracking_df: [0] # max range of motion angle [deg]
  tracking_pf: [90] # min range of motion angle [deg]
  mvc_df: [-1] # MVCT in dorsiflexion [Nm]
  mvc_pf: [1] # MVCT in plantarflexion [Nm]
  muscle_count: [4] # number of muscles for EMG recordings

#m1_x:
#  id: ['wristHRCEML']
#  left: false # flag for left or right limb
#  wrist: true # flag for wrist device
#  c0: [5.25] # coulomb friction constant of joint [Nm] 5
#  c1: [1.5] # viscous friction constant of joint [Nm/s] 1.5
#  c2: [0] # friction square root term
#  i_sin: [0.7] # inertia for sine term
#  i_cos: [0.0] # inertia for cosine term
#  t_bias: [-1.14] # theta bias term and angle from vertical position to lower joint limit [rad]
#  force_sensor_scale_factor: [0.074] # scale factor for force calibration
#  force_sensor_offset: [1500] # calibration offset for force sensor
#  ff_ratio: [0.75] # scale factor for feedforward compensation
#  kp: [2.28] # proportional term of PID controller
#  ki: [0.17] # integral term of PID controller
#  kd: [0.152] # derivative term of PID controller
#  vel_thresh: [6.0] # velocity threshold for using dynamic friction compensation [deg/s]
#  tau_thresh: [0.2] # torque threshold for using static friction compensation (not implemented) [Nm]
#  lowpass_cutoff_freq: [30.0] # cutoff frequency for lowpass interaction torque, angle and velocity filter [Hz]
#  motor_torque_cutoff_freq: [50.0] # cutoff frequency for lowpass motor torque command filter [Hz]
#  tick_max: [60.0] # counter for integral term reset [s]
#  tracking_offset: [72.48] # center of range of motion [deg]
#  tracking_df: [0] # max range of motion angle [deg]
#  tracking_pf: [90] # min range of motion angle [deg]
#  mvc_df: [-1] # MVCT in dorsiflexion [Nm]
#  mvc_pf: [1] # MVCT in plantarflexion [Nm]
#  muscle_count: [2] # number of muscles for EMG recordings

m1_x:
  id: ['ankleHRCEML']
  left: true # flag for left or right limb
  wrist: false # flag for wrist device
  c0: [2.6808] # coulomb friction constant of joint [Nm]
  c1: [2.2137] # viscous friction constant of joint [Nm/s]
  c2: [0] # friction square root term
  i_sin: [1.0] # inertia for sine term
  i_cos: [0.5] # inertia for cosine term
  t_bias: [-0.639] # theta bias term and angle from vertical position to lower joint limit [rad]
  force_sensor_scale_factor: [0.074] # scale factor for force calibration
  force_sensor_offset: [1500] # calibration offset for force sensor
  ff_ratio: [0.55] # scale factor for feedforward compensation
  kp: [1.08] # proportional term of PID controller
  ki: [0.17] # integral term of PID controller
  kd: [0] # derivative term of PID controller
  vel_thresh: [6.0] # velocity threshold for using dynamic friction compensation [deg/s]
  tau_thresh: [0.2] # torque threshold for using static friction compensation (not implemented) [Nm]
  lowpass_cutoff_freq: [30.0] # cutoff frequency for lowpass filter for interaction torque, angle and velocity [Hz]
  motor_torque_cutoff_freq: [50.0] # cutoff frequency for lowpass motor torque command filter [Hz]
  tick_max: [60.0] # counter for integral term reset [s]
  tracking_offset: [63] # center of range of motion [deg]
  tracking_df: [73] # max range of motion angle [deg]
  tracking_pf: [53] # min range of motion angle [deg]
  mvc_df: [-1] # MVCT in dorsiflexion [Nm]
  mvc_pf: [1] # MVCT in plantarflexion [Nm]
  muscle_count: [4] # number of muscles for EMG recordings